airbyte-cdk/java/airbyte-cdk/core/src/testFixtures/java/io/airbyte/cdk/testutils/TestDatabase.java

```diff
@@ -233,7 +233,6 @@ public B integrationTestConfigBuilder() {
   @Override
   public void close() {
     execSQL(this.cleanupSQL.stream());
-    dslContext.close();
```

Comment:
I bumped the version of `jooq` depended on by this version of the CDK and `DSLContext` is no longer an `AutoClosable` or a `Closable` for that matter.